teams/brno.yml

```diff
@@ -630,3 +630,35 @@
   links:
   - github-link: https://github.com/jpodivin
   - linkedin-link: https://www.linkedin.com/in/jiri-podivin-46601b156/
+
+- name: Veronika Konečná
```

Comment:
Veroniku už jednou v seznamu máme

Reply:
Aha! Divný, když jsem to psala, nenašla jsem ji, asi jsem tam měla překlep